### DISCUSSION TITLE
Twitter URL/Handle doesn't work

### DISCUSSION THREAD

**Discussion by safeboot** [elrumo/macOS_Big_Sur_icons_replacements#1286] [Help with the website] [answered]
On a user's page, their Twitter handle doesn't work, it redirects to **https://macosicons.com/u/safeboot1** instead of **https://twitter.com/safeboot1**.

That's about it.

**Comment by elrumo** [comment, marked as answer]
Thanks for pointing this out, it was a small bug where it would only work if you added your full Twitter URL. Should be fixed now.
